packages/esm-patient-chart-app/src/visit/start-visit-button.component.tsx

```diff
@@ -6,9 +6,10 @@ import { showSnackbar } from '@openmrs/esm-framework';
 
 interface StartVisitButtonProps {
   patientUuid: string;
+  handleBackToSearchList: () => void;
```

Comment:
Hi @Muppasanipraneeth the solution looks perfect, small nitpick shouldn't this new attribute be optional as the start visit button is used in other locations where it does not need to go back to search visit
```suggestion
  handleBackToSearchList?: () => void;
```